FlowCrypt/Controllers/Setup/SetupEKMKeyViewController.swift

```diff
@@ -36,7 +36,7 @@ final class SetupEKMKeyViewController: SetupCreatePassphraseAbstractViewControll
         self.keys = keys
         super.init(
             appContext: appContext,
-            user: UserId(email: appContext.user.email, name: appContext.user.name),
+            user: appContext.userId,
```

Comment:
`SetupCreatePassphraseAbstractViewController` initialiser doesn't need separate `user` param now, as we already pass `appContext` which contains `userId`